package.json

```diff
@@ -2,17 +2,30 @@
   "name": "@nextcloud/initial-state",
   "version": "2.0.0",
   "description": "Access data from the nextcloud server-side initial state API within apps.",
+  "homepage": "https://github.com/nextcloud/nextcloud-initial-state#readme",
+  "author": "Christoph Wurst",
```

Comment:
are you sure about that?

Reply:
I can undo this, it just "hurt my eye" that the package.json was like `meta data -> scripts -> metadata -> dependencies` :sweat_smile:

Reply:
I'm not actively maintaining this package so I think it would not make a lot of sense to list me as the author

Reply:
I did just move it, not added it :wink:
You already were mentioned as author.

Reply:
Oh